### ISSUE TITLE
Sinon

### ISSUE THREAD

**Issue by Fabio-Vicente** [simple-icons/simple-icons#8257]
### Brand Name

Sinon

### Website

https://sinonjs.org/

### Popularity Metric

https://www.similarweb.com/pt/website/sinonjs.org/#overview

### Official Resources for Icon and Color

https://sinonjs.org/assets/images/logo.png
color: #87543C, from the website theme

### Contributing

Yes

**Comment by ripzerotwo** [comment]
Fails to meet 500k requirement for website.

**Comment by ripzerotwo** [comment]
Before I continue, do you have any recommendations for the B & W logo? I will most likely have to recreate the logo as svg so yeah...

**Comment by Fabio-Vicente** [comment]
No, I don't. 😕 It's a very colorful icon and I have no idea how it could looks like on B & W...

**Comment by ripzerotwo** [comment]
No worries ill figure something out its just gonna take a bit.

**Comment by uncenter** [comment]
In scope with 5m+ weekly npm downloads and GitHub stars.
